sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-messaging-servicebus/pom.xml

```diff
@@ -5,14 +5,14 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.4.4</version>
+    <version>2.4.4</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-parent;external_dependency} -->
     <relativePath/> <!-- lookup parent from repository -->
   </parent>
   <groupId>com.azure.spring</groupId>
-  <artifactId>azure-spring-cloud-sample-servicebus-messaging</artifactId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-servicebus-messaging;current} -->
+  <artifactId>azure-spring-cloud-sample-messaging-servicebus</artifactId>
+  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-messaging-servicebus;current} -->
```

Comment:
should we also update the version_client.txt?

Reply:
Oh, seems like we don't have an entry there.

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt